gen3-cli/cot/command/query/__init__.py

```diff
@@ -125,12 +44,327 @@ def components_table(data):
+    required=True
+)
+@click.option(
+    '-i',
+    '--instance-id',
+    'instance_id',
+    required=True
+)
+@click.option(
+    '-v',
+    '--version-id',
+    'version_id',
+    required=True
+)
+@exceptions.handler()
+def describe_occurrence(ctx, **query_params):
+    """
+    Describes occurrence
+    """
+    if ctx.invoked_subcommand is not None:
+        ctx.obj['query_params'] = query_params
+        return
+    result = query_backend.run(
+        **ctx.obj['blueprint'],
+        cwd=os.getcwd(),
+        query_name='describe_occurrence',
+        query_params=query_params
+    )
+    click.echo(json.dumps(result, indent=4))
+
+
+@describe_occurrence.command('get')
+@click.argument('query_text')
+@click.pass_context
+@exceptions.handler()
+def describe_occurrence_get(ctx, query_text):
+    """
+    JMESPath subquery on the described occurrence data
+    """
+    result = query_backend.run(
+        **ctx.obj['blueprint'],
+        cwd=os.getcwd(),
+        query_name='describe_occurrence',
+        query_params=ctx.obj['query_params'],
+        sub_query_text=query_text
+    )
+    click.echo(json.dumps(result, indent=4))
+
+
+@describe_occurrence.command('attributes')
+@click.pass_context
+@exceptions.handler()
+@json_or_table_option(key_value_table)
+def describe_occurrence_attributes(ctx):
+    """
+    Describes occurrence attributes
+    """
+    result = query_backend.run(
+        **ctx.obj['blueprint'],
+        cwd=os.getcwd(),
+        query_name='describe_occurrence_attributes',
+        query_params=ctx.obj['query_params']
+    )
+    return result
+
+
+@describe_occurrence.command('solution')
+@click.pass_context
+@exceptions.handler()
+def describe_occurrence_solution(ctx):
+    """
+    Describes occurrence solution
+    """
+    result = query_backend.run(
+        **ctx.obj['blueprint'],
+        cwd=os.getcwd(),
+        query_name='describe_occurrence_solution',
+        query_params=ctx.obj['query_params']
+    )
+    click.echo(json.dumps(result, indent=4))
+
+
+@describe_occurrence.command('settings')
+@click.pass_context
+@exceptions.handler()
+def describe_occurrence_settings(ctx):
+    """
+    Describes occurrence settings
+    """
+    result = query_backend.run(
+        **ctx.obj['blueprint'],
+        cwd=os.getcwd(),
+        query_name='describe_occurrence_settings',
+        query_params=ctx.obj['query_params']
+    )
+    click.echo(json.dumps(result, indent=4))
+
+
+@describe_occurrence.command('resources')
+@click.pass_context
+@exceptions.handler()
+def describe_occurrence_resources(ctx):
+    """
+    Describes occurrence resources
+    """
+    result = query_backend.run(
+        **ctx.obj['blueprint'],
+        cwd=os.getcwd(),
+        query_name='describe_occurrence_resources',
+        query_params=ctx.obj['query_params']
+    )
+    click.echo(json.dumps(result, indent=4))
+
+
+@query_group.group('list')
+@click.pass_context
+def list_group(ctx):
+    """
+    Queries category
+    """
```

Comment:
and this; strange it's not a syntax error at all

Reply:
And why it should be a syntax error? Just curious because it's my preferred way to align docstrings.

Reply:
It just looks like some function without a body